modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -72,17 +68,24 @@ metadata:
       }]
     }'
 ----
-<1> `plugins` must be a list. The first item in the list must be secondary network underpinning the VRF network. The second item in the list is the VRF plugin configuration.
+<1> `plugins` must be a list. The first item in the list must be the secondary network underpinning the VRF network. The second item in the list is the VRF plugin configuration.
 <2> `type` must be set to `vrf`.
 <3> `vrfname` is the name of the VRF that the interface is assigned to. If it does not exist in the pod, it is created.
-<4> `table` is the routing table ID. Optional. By default, the `tableid` parameter is used. If it is not specified, the CNI assigns a free routing table ID to the VRF.
+<4> `table` is the (optional) routing table ID. By default, the `tableid` parameter is used. If it is not specified, the CNI assigns a free routing table ID to the VRF.
 +
 [NOTE]
 ====
 VRF will function correctly only when the resource is of type `netdevice`.
```

Comment:
Suggested edit, since this is existing feel free to ignore: 

s/VRF will function/VRF functions